src/main/java/dev/jbang/cli/Edit.java

```diff
@@ -249,52 +240,52 @@ File createProjectForEdit(Script script, boolean reload) throws IOException {
 
 		// both collectDependencies and repositories are manipulated by
 		// resolveDependencies
-		List<MavenRepo> repositories = script.getRepositories();
+		List<MavenRepo> repositories = script.script().collectAllRepositories();
```

Comment:
why this rename ? before collect* was for non-cached and get* was caching collected?

Reply:
Ok, that distintion must have been lost at some point, because we definitely have `collectXxxx()` calls that do caching right now. Not seeing and not knowing the difference I just decided to unify the names. But I can definitely change them all to `getXxxx()`.

Reply:
Done